server/lib/channel.dart

```diff
@@ -1,38 +1,44 @@
 import 'server.dart';
+import 'service/google_api.dart';
 
-/// This type initializes an application.
-///
-/// Override methods in this class to set up routes and initialize services like
-/// database connections. See http://aqueduct.io/docs/http/channel/.
+/// A subclass of [ApplicationChannel] that is created for each isolate.
 class ServerChannel extends ApplicationChannel {
-  /// Initialize services in this method.
-  ///
-  /// Implement this method to initialize services, read values from [options]
-  /// and any other initialization required before constructing [entryPoint].
-  ///
-  /// This method is invoked prior to [entryPoint] being accessed.
+  /// The [GoogleApi] instance that provides authenticated clients to access
+  /// Google APIs.
+  GoogleApi googleApi;
+
+  ServerConfiguration configuration;
+
+  /// Initializes logger, configuration values, and services to be injected.
   @override
   Future prepare() async {
-    logger.onRecord.listen(
-        (rec) => print("$rec ${rec.error ?? ""} ${rec.stackTrace ?? ""}"));
+    // Record stack traces if level is severe or higher.
+    recordStackTraceAtLevel = Level.SEVERE;
+    logger.onRecord.listen((final record) {
+      print(record);
+      print(record.error ?? '');
+      print(record.stackTrace ?? '');
+    });
+    // Uncomment for stack trace in response.
+    // Controller.includeErrorDetailsInServerErrorResponses = true;
+
+    configuration = ServerConfiguration(options.configurationFilePath);
+
+    // Initialize services.
+    googleApi = GoogleApi(configuration.clientId, configuration.clientSecret);
   }
 
-  /// Construct the request channel.
-  ///
-  /// Return an instance of some [Controller] that will be the initial receiver
-  /// of all [Request]s.
-  ///
-  /// This method is invoked after [prepare].
   @override
   Controller get entryPoint {
     final router = Router();
-
-    // Prefer to use `link` instead of `linkFunction`.
-    // See: https://aqueduct.io/docs/http/request_controller/
-    router.route("/example").linkFunction((request) async {
-      return Response.ok({"key": "value"});
-    });
-
     return router;
   }
 }
+
+class ServerConfiguration extends Configuration {
+  String clientId;
+  String clientSecret;
+  APIConfiguration displayVideo360;
```

Comment:
Be consistent to use `Api` or `API`

Reply:
Can we just use Api then?

go/robtierney-java-readability-snippets#make-acronyms-camelcased-instead-of-fully-uppercased

Reply:
Sorry, [APIConfiguration](https://pub.dev/documentation/safe_config/latest/safe_config/APIConfiguration-class.html) wasn't chosen by me. I'll be consistent with `Api` though, sure.

Reply:
Oh I see. nvm then.